src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/MsQuicSession.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net.Quic.Implementations.MsQuic.Internal
+{
+    internal sealed class MsQuicSession : IDisposable
+    {
+        private bool _disposed = false;
+        private IntPtr _nativeObjPtr;
+        private MsQuicApi _api;
+        private bool _opened;
+
+        internal MsQuicSession()
+        {
+            _api = MsQuicApi.Api;
```

Comment:
Does this need to be a field? Seems like we could just use MsQuicApi.Api directly.